pkg/auth/auth.go

```diff
@@ -300,6 +300,9 @@ func IsAdminClient(ctx context.Context) bool {
 var AdminUserExists func(ctx context.Context) bool
 var CreateAdminUser func(ctx context.Context) (string, string, error)
 
+var ErrServerAuthDisabled = status.Errorf(
```

Comment:
formatting function at non formatting case , please use formatting string.

Reply:
@volknanebo i replaced `Errorf` with `Error` - is supposed that's what you meant?

Reply:
+